cmd/oras/root/attach.go

```diff
@@ -164,7 +169,19 @@ func runAttach(ctx context.Context, opts attachOptions) error {
 
 	root, err := pushArtifact(dst, pack, copy)
 	if err != nil {
-		return err
+		if opts.strict {
+			return err
+		}
+		var re *remote.ReferrersError
+		if !errors.As(err, &re) || !re.IsReferrersIndexDelete() {
+			// not referrer index delete error
+			return err
+		}
+		var errResp *errcode.ErrorResponse
+		if !errors.As(re.Err, &errResp) || errResp.StatusCode != http.StatusMethodNotAllowed {
+			return err
+		}
+		logger.Info("Attach is successful but removal of outdated referrers index from remote registry failed. Garbage collection may be required.")
```

Comment:
Is this an `Info` or `Warn`?

Reply:
Should user see this log without `--verbose`?

 - yes => it's `Info`
 - no => it's `Warn`

Reply:
Should that the other way around? `--verbose` toggles `Info` level and by default it is `Warn` level.

Reply:
Anyway, this log should be printed even if `--verbose` is not toggled.